modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -139,7 +139,10 @@ export default class NestedTabs extends Base {
 			$activeTitle = this.elements.$tabTitles.filter( activeTitleFilter ),
 			$activeContent = this.elements.$tabContents.filter( activeContentFilter );
 
-		$activeTitle.add( $activeContent ).removeClass( activeClass );
+		// $activeTitle.add( $activeContent ).removeClass( activeClass );
+		$activeContent.removeClass( activeClass );
+		setTimeout( $activeTitle.removeClass( activeClass ), 50 );
+
```

Comment:
Adding and removing the `activeClass` for the `title` element after the `content` element seems to solve our problem as well.